lib/irb.rb

```diff
@@ -1132,7 +1132,7 @@ def build_statement(code)
         return Statement::EmptyInput.new
       end
 
-      code.force_encoding(@context.io.encoding)
+      code.dup.force_encoding(@context.io.encoding)
```

Comment:
This statement has no effect. We need to change the encoding of `code`. 
maybe `code = code.dup.force_encoidng(...)`

Reply:
Thanks for your review and suggestion! I fixed them at 
37fe153.

By the way, I met the warning in [SciRuby/iruby's test code](https://github.com/SciRuby/iruby/blob/a19e3f3cd2e95553b5ea463fc8e4b35d72c6eef4/test/iruby/backend_test.rb#L13
), [SciRuby/iruby's code](https://github.com/SciRuby/iruby/blob/f341266ea123a0a916b17b148742bb101b6c0a41/lib/iruby/backend.rb#L60).